snap/snapcraft.yaml

```diff
@@ -21,6 +21,25 @@ slots:
     bus: system
     name: io.openthread.BorderRouter.wpan0
 
+plugs:
+  system-etc-iproute:
+    interface: system-files
+    write:
+      - /etc/iproute2
+  system-etc-sysctl:
+    interface: system-files
+    write:
+      - /etc/sysctl.conf
+      - /etc/sysctl.d
+  system-run-openthread-wpan0:
+    interface: system-files
+    write:
+      - /run/openthread-wpan0.lock
+      - /run/openthread-wpan0.sock
+  system-run:
+    interface: system-files
+    write:
+      - /run
```

Comment:
This is never used. Should be added where access to `/run/openthread-<if>.sock` is needed and used only if PAN interface isn't `wpan0`.

Reply:
Great! As `system-run` is not used, and it grants too much access for otbr-agent, I added a section in the README on how users could add `system-run-openthread-<thread-if>` whenever needed. This way, it is safer and avoids the exposure of the entire /run directory.

Reply:
The user should not be expected to rebuild the snap to install it. How will they receive updates? The installation should be done by downloading from the store, where the snap is signed and kept up to date.